File: src/main/java/fialka/contoller/Controller.java
package fialka.contoller;

import fialka.entity.Person;
import fialka.repository.DBRepository;
import fialka.utils.Gender;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping(value = "/test")
public class Controller {

    @Autowired
    private DBRepository dbRepository;

    private static ResponseEntity okResponse = new ResponseEntity(HttpStatus.OK);

    @GetMapping("/ping")
    public ResponseEntity ping() {
        return new ResponseEntity<String>("blabla", HttpStatus.OK);
    }

    @PostMapping("/ping2")
    public ResponseEntity ping2() {
        return okResponse;
    }

    @PostMapping("/insertToDb")
    public ResponseEntity insertToDb() {
        dbRepository.insertToDB(new Person("Gurbanguly", "Berdymukhamedov", Gender.Male));
        return okResponse;
    }

}

File: hibernate-telusko/src/main/kotlin/fialka/Controller.kt
package fialka

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping(value = ["/test"])
class Controller {

    @Autowired
    private lateinit var service: Service

    @GetMapping("/ping")
    fun ping(): String? {
        return "blabla"
    }

    @PostMapping("/ping2")
    fun ping2() {
        service.save()
    }
}